lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1532,6 +1533,21 @@ void SwiftASTContext::ApplyWorkingDir(
   clang_argument.append(joined_path.begin(), joined_path.end());
 }
 
+void SwiftASTContext::ApplyDiagnosticOptions() {
+  const auto &opts = GetCompilerInvocation().getDiagnosticOptions();
+  if (opts.PrintDiagnosticNames) {
```

Comment:
[LLVM style is to omit braces on simple single-statements](https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements)

Reply:
Braces removed